Research/ResearchUI/iOS/RSDStepViewController.swift

```diff
@@ -84,6 +84,9 @@ open class RSDStepViewController : UIViewController, RSDStepController, RSDCance
             return result
         }
     }()
+
+    /// Property for injecting custom view configuration, used for UI testing.
+    public var configureOnViewDidLoad: ((_ viewController: RSDStepViewController) -> Void)? = nil
```

Comment:
the comment helps, but even so, do you think the property name should reflect the fact that it's only used in testing to prevent confusion? maybe something like `configureForTesting` or `configureForTestOnLoad`?

also, less importantly, i feel like the `viewController` param label isn't really needed/doesn't provide additional info (although this could just be a personal preference thing, in which case feel free to disregard!)

Reply:
There's more discussion on the naming taking place on the other PR.  Leaning more towards the underscore-separated names that clearly separate "what the closure is" from "where the closure gets invoked".  

Any thoughts/preferences between these?

```
    public var injectViewConfig_onViewDidLoad: ((_ viewController: RSDStepViewController) -> Void)? = nil
```

```
    public var inject_onViewDidLoad: ((_ viewController: RSDStepViewController) -> Void)? = nil
```

```
    public var callback_onViewDidLoad: ((_ viewController: RSDStepViewController) -> Void)? = nil
```